Aptoide/aptoide-client-v8
# [ASV-1035] New Comments Core and Store implementation

**What does this PR do?**

   This pull-request has the core of the comments API using clean architecture. This core will be the dependency for every view that touches comments in the future. This also has a low def implementation of the comments with the new look and feel in the Stores ( stores comments ) and it is composed in 3 main views: the storefragment (still widgets-displayables), which has the store's latests comments, the commentsfragment (generic list of comments) and commentsdetailfragment (generic comment detail with latest 10 replies).

Also tried to test most of the flows.

**Database changed?**

   No

**Where should the reviewer start?**

- [ ] CommentsFragment.java
- [ ] StoreLatestCommentsWidget.java
- [ ] CommentDetailFragment.java

**How should this be manually tested?**

  Go to a store with comments, see comments, click more, see all comments, make a reply, pull to refresh, infinite scroll, click comment with replies, try to reply etc.

**What are the relevant tickets?**

  Tickets related to this pull-request: [ASV-1042](https://aptoide.atlassian.net/browse/ASV-1042)
[ASV-1043](https://aptoide.atlassian.net/browse/ASV-1043)
[ASV-1044](https://aptoide.atlassian.net/browse/ASV-1044)

**Code Review Checklist**
